blog/themes/3mdeb/layouts/partials/header.html

```diff
@@ -1,113 +1,136 @@
+        </ul>
+      </li>
+      <li class="dropdown"><a href="https://3mdeb.com/services/">SERVICES</a>
+        <ul class="submenu">
+          <li><a href="https://3mdeb.com/services/embedded-firmware/">Embedded Firmware</a></li>
+          <li><a href="https://3mdeb.com/services/operating-systems/">Operating Systems</a></li>
+          <li><a href="https://3mdeb.com/services/hypervisors/">Hypervisors</a></li>
+          <li><a href="https://3mdeb.com/services/software-and-hardware-security/">Software and Hardware Security</a>
+          </li>
+          <li><a href="https://3mdeb.com/services/hardware-support/">Hardware Support</a></li>
+          <li><a href="https://3mdeb.com/services/internet-of-things-solutions/">Internet of Things Solutions</a></li>
+          <li><a href="https://3mdeb.com/services/automated-validation/">Automated Validation</a></li>
+          <li><a href="https://3mdeb.com/services/service-catalogues/">Services Catalogues</a></li>
+        </ul>
+      </li>
+      <li class="dropdown"><a href="https://3mdeb.com/news/">NEWS</a>
+        <ul class="submenu">
+          <li><a href="https://blog.3mdeb.com/">Blog</a></li>
+          <li><a href="https://3mdeb.com/news/events/">Events</a></li>
+          <li><a href="https://3mdeb.com/news/press-releases/">Press Releases</a></li>
+          <li><a href="https://opensource.3mdeb.com/">Our Contribution</a></li>
+        </ul>
+      </li>
+      <li><a href="https://shop.3mdeb.com/">SHOP</a></li>
+      <li><a href="https://3mdeb.com/contact/">CONTACT</a></li>
+    </ul>
+  </div>
+  </header>
 
-    <!-- hero -->
-    <section class="hero is-info is-medium">
-        <div class="hero-body" style="background-image: url({{ "img/bg-pattern.jpg" | absURL }});">
-            <div class="container has-text-centered">
-                <br>
-                <h1 class="title is-size-1">
-                    {{ if eq .Type "tags" }}
-                        #{{ .Title }}
-                    {{ else if eq .Type "archives" }}
-                        Archive {{ .Title }}
-                    {{ else if eq .Type "authors" }}
-                        {{ if isset .Params "name" }}
-                          {{ .Params.name }}
-                        {{ else }}
-                          Authors
-                        {{ end }}
-                    {{ else }}
-                        {{ .Title }}
-                    {{ end }}
-                </h1>
-                {{ if eq .Type "article" }}
-                    {{ .Date.Format "January 2, 2006" }} &middot;&nbsp;{{ .ReadingTime }} min read
-                {{ end }}
-            </div>
-        </div>
-    </section>
+  <!-- hero -->
+  <section class="hero is-info is-medium">
+    <div class="hero-body" style="background-image: url({{ "img/bg-pattern.jpg" | absURL }});">
+      <div class="container has-text-centered">
+        <br>
+        <h1 class="title is-size-1">
+          {{ if eq .Type "tags" }}
+          #{{ .Title }}
```

Comment:
@rg3mdeb please give me a reason why you have removed the indentation within hugo syntax?